api/build.gradle

```diff
@@ -6,7 +6,7 @@ dependencies {
 
     compile("com.paypal.sdk:rest-api-sdk:+")
     compile("org.projectlombok:lombok:1.16.20")
-    compile("org.postgresql:postgresql:9.3-1100-jdbc4")
+    compile("org.postgresql:postgresql:42.1.4")
```

Comment:
Does this driver connect to the rds instance in aws? We can test by starting the app up and changing the config file to point to the RDS url with the correct username and pass

Reply:
Just tested this. It works.